resources/helm/overlays/istio_cni/templates/daemonset-v2.3.yaml

```diff
@@ -0,0 +1,128 @@
+{{ if .Values.cni.enabled }}
+{{- if and (.Values.cni.supportedReleases) (eq .Values.cni.instanceVersion "v2.3") }}
+# This manifest installs the Istio install-cni container, as well
+# as the Istio CNI plugin and config on
+# each master and worker node in a Kubernetes cluster.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: istio-cni-node-v2.3
```

Comment:
Do we want to call this `istio-cni-node-v2-3` instead so that the name and the `k8s-app` label value are consistent?